scripts/generate_json_docs.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
```

Comment:
To match the gcloud-node navigation style which has "Thing » Subthing", the interpreter was complaining, but I could probably define it on the string itself. I'll try it out!